analyses/cell-type-wilms-tumor-06/00_run_workflow.R

```diff
@@ -85,5 +85,16 @@ for (sample_id in metadata$scpca_sample_id) {
   }
 }
 
+for (sample_id in metadata$scpca_sample_id) {
+ if (!running_ci) {
+    # Cluster exploration
+    rmarkdown::render(input = file.path(notebook_template_dir, "03_clustering_exploration.Rmd"),
+                      params = list(scpca_project_id = project_id, sample_id = sample_id),
+                      output_format = "html_document",
+                      output_file = paste0("03_clustering_exploration", sample_id, ".html"),
+                      output_dir = file.path(notebook_output_dir, sample_id))
+  }
+}
+
```

Comment:
Since this for-loop is actually the same as the previous for loop (which also contains this if-statement for the label transfer notebooks) `for (sample_id in metadata$scpca_sample_id) {`, you should be able to move this code up into the previous for-loop and get rid of this additional for-loop.

In other words, the cluster exploration step can just go right after the step where you render this notebook: `02b_label-transfer_fetal_kidney_reference_Stewart.Rmd`.